Miki.Framework/Extension/DiscordExtensions.cs

```diff
@@ -129,26 +128,18 @@ public static async Task<IDiscordMessage> SendToChannel(this DiscordEmbed embed,
 				{
 					if(!string.IsNullOrEmpty(embed.Image?.Url ?? ""))
 					{
-						using(WebClient wc = new WebClient())
-						{
-							byte[] image = wc.DownloadData(embed.Image.Url);
-							using(MemoryStream ms = new MemoryStream(image))
-							{
-								return await channel.SendFileAsync(ms, "output.png", embed.ToMessageBuilder().Build());
-							}
-						}
-					}
+                        using WebClient wc = new WebClient();
```

Comment:
Maybe we need to replace this with an HttpClient in the future.

Quote by Microsoft:
> We don't recommend that you use the WebClient class for new development. Instead, use the System.Net.Http.HttpClient class.
>
> https://docs.microsoft.com/en-us/dotnet/api/system.net.webclient#remarks

And while we are at it, use the [IHttpClientFactory](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests).